package.json

```diff
@@ -2,16 +2,16 @@
   "name": "hackerman-bot",
   "version": "1.0.0",
   "description": "Hackerman bot",
-  "main": "index.js",
+  "main": "Bot.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js"
+    "start": "node Bot.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Ubspy/Hackerman-bot.git"
   },
-  "author": "Jack Moren",
+  "author": "Jack Moren, drhacko",
```

Comment:
I'd use your real name there, but you don't gotta

Reply:
you right `Dylan Hack`